### DISCUSSION TITLE
Autosize column with each value

### DISCUSSION THREAD

**Discussion by andywijaya15** [spatie/simple-excel#150] [Q&A]
how to autosize column in export ?
my data longer than my header,
i can do it with laravel-excel with this
![image](https://github.com/spatie/simple-excel/assets/79361187/931b7038-743b-4f9a-af75-829d5bfa20d0)
but laravel-excel is too much for my simple export import,
how do it in this package ?
thank youu

**Comment by SnakeO** [comment]
Any word on this?
